tools/create_testing_volumes.py

```diff
@@ -73,6 +73,7 @@ def mount_volumes():
 export MESOS_WORK_DIR MESOS_RESOURCES
 eval $(sed -E "s/^([A-Z_]+)=(.*)$/\\1='\\2'/" /opt/mesosphere/etc/mesos-slave-common)  # Set up `MESOS_WORK_DIR`.
 eval $(sed -E "s/^([A-Z_]+)=(.*)$/\\1='\\2'/" /opt/mesosphere/etc/mesos-slave)         # Set up `MESOS_RESOURCES`.
+source /opt/mesosphere/etc/mesos-slave-common
```

Comment:
I will remove this part of the diff, this was done when i am experimenting with dcos-launch stuff.